components/PlantCard.js

```diff
@@ -0,0 +1,41 @@
+import styled from "styled-components";
+import Link from "next/link";
+
+export default function PlantCard({ plant }) {
+  return (
+    <Container>
+      <h2>{plant.name}</h2>
+      <StyledParagraph>{plant.botanicalName}</StyledParagraph>
+      <RoundImage
+        alt={`Image of ${plant.name}`}
```

Comment:
I know i suggested something similar... and this is really nitpicking now. But i guess we would just use `plant.name`

> Redundant alt attribute. Screen-readers already announce `img` tags as an image. You don’t need to use the words `image`, `photo,` or `picture` (or any specified custom words) in the alt prop.